sp800-63-3/sec1_2_introduction.md

```diff
@@ -68,21 +68,18 @@ their authentication assurance requirements:
     level* - As some implementations may create or compound particular
     risks, agencies should conduct a final validation to confirm that
     the system achieves the required assurance level for the
-    user-to-agency process. NIST SP 800-53A [[SP 800-53A]](#SP800-53A)
+    user-to-agency process. NIST [[SP 800-53A]](https://pages.nist.gov/800-63-3/sp800-63-3.html#SP800-53A)
     provides requirements for the assessment of the implemented system
     during the validation process. Validation should be performed as
-    part of a security authorization process as described in NIST SP
-    800-37, Revision 1 [[SP 800-37]](#SP800-37).
+    part of a security authorization process as described in NIST [[SP 800-37]](https://pages.nist.gov/800-63-3/sp800-63-3.html#SP800-37), Revision 1.
```

Comment:
i'm going to stop commenting on each of these.  Please make sure the above applies for all.